### ISSUE TITLE
[3.0b1 issue] gr.Tabs() and gr.TabItem() don’t work inside of a gr.Column() or a gr.Row()

### ISSUE THREAD

**Issue by apolinario** [gradio-app/gradio#1247]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

gr.Tabs() and gr.TabItem() don’t work inside of a gr.Column() or a gr.Row() since 2.9b24 (it worked before that). I hope it`s a bug because I think it should work otherwise a design like [MindsEye Lite](https://huggingface.co/spaces/multimodalart/mindseye-lite) is not possible (where on the left you have multiple tabs and on the right the output)

### Reproduction

```
import gradio as gr

def update(name):
    return f"Welcome to Gradio, {name}!"

demo = gr.Blocks()

with demo:
    gr.Markdown("Start typing below and then click **Run** to see the output.")
    with gr.Row():
        with gr.Column():
            with gr.Tabs():
                with gr.TabItem("Image output"):
                    inp = gr.Textbox(placeholder="What is your name?")
                    btn = gr.Button("Run")
            
        with gr.Column():
            out = gr.Textbox()
        
    btn.click(fn=update, inputs=inp, outputs=out)

demo.launch()
```

### Screenshot

![image](https://user-images.githubusercontent.com/788417/168314650-6456e9ee-0d92-4d50-8ec4-31cdbc94bc42.png)
_works on 2.9b24_

![image](https://user-images.githubusercontent.com/788417/168314848-21cf0295-a76c-4d3f-9758-d42784a76026.png)
_doesn't work on 3.0b1_

### System Info

```shell
Gradio==3.0b1
All systems
```

### Severity

annoyance

**Comment by pngwn** [comment]
I think this is the same issue as #1237 and should be fixed by #1246.  We'll create a new beta release later today to get this out.

**Comment by abidlabs** [comment]
Works now with `gradio==3.0b5`!
